cookbooks/aws-parallelcluster-slurm/spec/unit/libraries/get_target_group_name_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../../libraries/helpers'
```

Comment:
Is this import being used?

Reply:
Yes. The `get_target_group_name` function is in `helpers.rb`